### DISCUSSION TITLE
30,000 Sats for Prepay on failed Loop Out?

### DISCUSSION THREAD

**Discussion by ElChopChop** [Ride-The-Lightning/RTL#855] [Q&A] [answered]
I've tried to Loop Out but it's failed and appears to have cost me 30,000 Sats for "Cost Server". Is this right? I thought that was considered a Prepay amount and was expecting it back, especially as the transaction failed...

![Screenshot_20211110-103127_Brave](https://user-images.githubusercontent.com/94048277/141097278-38e16fa8-8a86-4b70-a89d-a276c3db87f2.jpg)

**Comment by saubyk** [comment]
Hi, @ElChopChop If the loop out failed, you should not be charged anything.

Please verify if your node actually made any payment for the 30,000 sats shown here.

**Reply by ElChopChop** [reply]
Hi @saubyk yes I've been charged 30,000 sats for each of my loop outs, one of which has failed and the other is still in "initiated" status since 8th Nov.
![Screenshot_20211111-061434_Brave](https://user-images.githubusercontent.com/94048277/141248380-e54d43ac-3649-4079-98cf-da7cc9dabd7a.jpg)

**Reply by ElChopChop** [reply]
@saubyk The second Loop Out also failed and I was also charged 30k Sats. Anything I can do about this? Both of the 30k payments were paid from my node with a reference of "Prepay".

![Loop Out 60k Sats](https://user-images.githubusercontent.com/94048277/144751414-7b9b567e-5ab3-494d-9ed3-ae51da4474f8.PNG)

**Comment by ElChopChop** [comment]
This is odd, I've definitely been charged 30,000 sats for the failed Loop Out and also 30,000 sats for the other Loop Out which has been in "Initiated" state for over a week. Anyone have any suggestions for me?

![image](https://user-images.githubusercontent.com/94048277/141759271-d09fe10f-faef-467e-b7d2-fe545f791f63.png)

**Comment by saubyk** [comment, marked as answer]
hello @ElChopChop this question should be addressed to Lightning Labs team. You can open an issue on the Loop GitHub:
https://github.com/lightninglabs/loop/issues

They only can provide a response on why a fee is being charged for the failed swaps. Thanks
